src/manage_sql.c

```diff
@@ -32603,14 +32603,20 @@ parse_osp_report (task_t task, report_t report, const char *report_xml)
           /* TODO: This should probably be handled by the "Host Detail"
            *        result type with extra source info in OSP.
            */
-          if (manage_report_host_detail (global_current_report,
-                                         host,
-                                         desc))
+          if (manage_report_host_detail (report, host, desc))
             g_warning ("%s: Failed to add report detail for host '%s': %s",
                       __FUNCTION__,
                       host,
                       desc);
         }
+      else if (host && nvt_id && desc && (strcmp (nvt_id, "HOST_START") == 0))
```

Comment:
This is a strange feature of OSP.

Reply:
I also think it would be a good idea to change this. How would you handle the host start and end times in OSP?
My suggestion would be to add a new host detail result type to OSP and use "host_start" and "host_end" as the names.
I'm not sure if we should do it in this PR already.